blueocean-config/src/test/java/io/jenkins/blueocean/config/FeaturesTest.java

```diff
@@ -0,0 +1,21 @@
+package io.jenkins.blueocean.config;
+
+import com.google.common.collect.ImmutableMap;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class FeaturesTest {
+    @Test
+    public void testLookupFeature() {
+        Features features = new Features();
+        assertEquals("false", features.getFeature("my.cool.feature"));
+    }
+
+    @Test
+    public void testSystemPropertyOverride() {
+        Features features = new Features();
+        System.setProperty("feature.my.cool.feature", "true");
```

Comment:
@i386 features.properties should have been loaded from main source package correctly. But maybe you want to test custom properties.